packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/NotificationSerializer.java

```diff
@@ -192,30 +221,55 @@ private static String getChannelId(NotificationTrigger trigger) {
     return null;
   }
 
-    @NotNull
-    public static Bundle toResponseBundleFromExtras(Bundle extras) {
-      Bundle serializedContent = new Bundle();
-      serializedContent.putString("title", extras.getString("title"));
+  @NotNull
+  public static Bundle toResponseBundleFromExtras(Bundle extras) {
+    Bundle serializedContent = new Bundle();
+    serializedContent.putString("title", extras.getString("title"));
+    String body = extras.getString("body");
+    String projectId = extras.getString("projectId");
```

Comment:
If possible, don't special case for EAS Notify. Instead, special case for "JSON-like bodies".